src/cascadia/TerminalSettings/KeyChord.cpp

```diff
@@ -4,6 +4,82 @@
 #include "pch.h"
 #include "KeyChord.h"
 
+static const std::wstring CTRL_KEY{ L"ctrl" };
+static const std::wstring SHIFT_KEY{ L"shift" };
+static const std::wstring ALT_KEY{ L"alt" };
+
+static const std::vector<std::pair<int32_t, std::wstring>> vkeyNamePairs {
+    { VK_BACK       , L"backspace"},
+    { VK_TAB        , L"tab"},
+    { VK_RETURN     , L"enter" },
+    { VK_ESCAPE     , L"esc" },
+    { VK_SPACE      , L"space" },
+    { VK_PRIOR      , L"pgup" },
+    { VK_NEXT       , L"pgdn" },
+    { VK_END        , L"end" },
+    { VK_HOME       , L"home" },
+    { VK_LEFT       , L"left" },
+    { VK_UP         , L"up" },
+    { VK_RIGHT      , L"right" },
+    { VK_DOWN       , L"down" },
+    { VK_INSERT     , L"insert" },
+    { VK_DELETE     , L"delete" },
+    { VK_NUMPAD0    , L"numpad_0" },
+    { VK_NUMPAD1    , L"numpad_1" },
+    { VK_NUMPAD2    , L"numpad_2" },
+    { VK_NUMPAD3    , L"numpad_3" },
+    { VK_NUMPAD4    , L"numpad_4" },
+    { VK_NUMPAD5    , L"numpad_5" },
+    { VK_NUMPAD6    , L"numpad_6" },
+    { VK_NUMPAD7    , L"numpad_7" },
+    { VK_NUMPAD8    , L"numpad_8" },
+    { VK_NUMPAD9    , L"numpad_9" },
+    { VK_MULTIPLY   , L"*" },
+    { VK_ADD        , L"+" },
+    { VK_SUBTRACT   , L"-" },
+    { VK_DECIMAL    , L"." },
+    { VK_DIVIDE     , L"/" },
+    { VK_F1         , L"f1" },
+    { VK_F2         , L"f2" },
+    { VK_F3         , L"f3" },
+    { VK_F4         , L"f4" },
+    { VK_F5         , L"f5" },
+    { VK_F6         , L"f6" },
+    { VK_F7         , L"f7" },
+    { VK_F8         , L"f8" },
+    { VK_F9         , L"f9" },
+    { VK_F10        , L"f10" },
+    { VK_F11        , L"f11" },
+    { VK_F12        , L"f12" },
+    { VK_F13        , L"f13" },
+    { VK_F14        , L"f14" },
+    { VK_F15        , L"f15" },
+    { VK_F16        , L"f16" },
+    { VK_F17        , L"f17" },
+    { VK_F18        , L"f18" },
+    { VK_F19        , L"f19" },
+    { VK_F20        , L"f20" },
+    { VK_F21        , L"f21" },
+    { VK_F22        , L"f22" },
+    { VK_F23        , L"f23" },
+    { VK_F24        , L"f24" },
+    { VK_OEM_PLUS   , L"+" },
```

Comment:
how does one cay `ctrl++`

Reply:
oh frick yea, that's why sublime had `keypad_plus` as the name. I'll fix that.

Reply:
`keypad_plus` is actually a different plus (!!) than `shift =` :(

Reply:
technically the key binding here is ctrl+shift+equals (aka "plus")